backend/routes/doctorOrders.js

```diff
@@ -25,7 +25,7 @@ router.patch("/fhir/rems/:id", async (req, res, next) => {
   });
 });
 
-router.patch("/fhir/rems/pickedUp/:id", async (req, res, next) => {
+router.patch("/$process-message/pickedUp/:id", async (req, res, next) => {
```

Comment:
change this to be ([\$])process-message. See https://github.com/expressjs/express/issues/2987 for more context